Add post about changing how we ask about disabilities and health conditions (#758)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2022-11-09-changing-how-we-ask-about-disabilities-and-health-conditions.md

![Screenshot showing page asking ’Are you disabled?’ with the options ‘Yes’, ‘No’ and ‘Prefer not to say’](are-you-disabled.png "Previous design for filter question")

If a candidate selected Yes, then we presented a list of checkboxes, with the question ‘What disabilities do you have?’

![Screenshot showing page with the question ‘What disabilities do you have?’ and this list of options: Blind or a serious visual impairment not corrected by glasses, Dear or a serious hearing impairment, Learning difficulty for example dyslexia, Long-standing illness for example cancer, Mental health condition for example depression, Physical disability or mobility issue for example impaired use of arms or legs, Social or communication impairment for example Asperger’s and Other.](disabilities-before.png "Previous design for disabilities question")

The main issue with presenting the questions in this way is that some candidates who might have selected one or more of the checkboxes may not have done so as they do not consider themselves disabled. We had some evidence of this from user research sessions, where one user who was dyslexic commented that not everyone considers this a disability.

> In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority updated their recommended list of options to present.

Reviewer: ```suggestion
In addition, the Higher Education Statistics Agency have also updated their recommended list of disability options for the 2022 to 2023 recruitment cycle.
```

## What we changed

![Screenshot showing page with the question ‘Do you have any of these disabilities or health conditions?’](disabilities-and-health-conditions.png "Updated design for disabilities or health conditions question")

We removed the first question, and instead present all users with a list of checkboxes.

We changed the question to ‘Do you have any of these disabilities or health conditions?’ and added a ‘I do not have any of these disabilities or health conditions’ option.

We added some content at the top of the page to confirm that a candidate’s answer is not shown to providers when they are deciding whether or not to offer a place on a course, and instead is only revealed if an offer has been accepted.

We also added a link back to the section of the application where candidates can ask for support if they are disabled as it is important that users do not expect providers to consider their answer to this question at the interview or assessment stage.

We reviewed the content for each of the options, with the aim to front-load the most recognisable words and phrases, and to avoid using hint text except to give examples.

## Alternatives we considered

> The Higher Education Statistics Authority has some [guidance on collecting type of disability for initial teacher training](https://www.hesa.ac.uk/collection/c22053/e/disable) which includes suggested question wording from [Advance HE](https://www.advance-he.ac.uk). Their suggestion is to include a lengthy explanation of disability as defined in the Equality Act 2010, and then to ask this question:

Reviewer: ```suggestion
The Higher Education Statistics Agency has some [guidance on collecting type of disability for initial teacher training](https://www.hesa.ac.uk/collection/c22053/e/disable) which includes suggested question wording from [Advance HE](https://www.advance-he.ac.uk). Their suggestion is to include a lengthy explanation of disability as defined in the Equality Act 2010, and then to ask this question:
```

> Considering the above, do you have an impairment, health condition or learning difference that has substantial and long-term impact on your ability to carry out normal day-to-day activities?

We considered that this was too much of a cognitive burden on candidates, particularly at the application stage of the process.

The [equality information pattern](https://design-system.service.gov.uk/patterns/equality-information/) in the GOV.UK Design System asks an initial question of ‘Do you have any physical or mental health conditions or illness lasting or expected to last 12 months or more?’ and then a follow-up question of ‘Do any of your conditions or illnesses reduce your ability to carry out day to day activities?’. This pattern does not include a list of disability or health conditions.

## Initial results

> Comparing the first 28 days of the 2022 to 2023 recruitment cycle to the first 28 days of the previous cycle (around 8,000 applications), 3 times more candidates have selected one or more disability or health condition (15% compared to 4%).

Reviewer: ```suggestion
Comparing the first 28 days of the 2022 to 2023 recruitment cycle to the first 28 days of the previous cycle (around 8,000 applications), 15% of candidates have selected one or more disability or health condition compared to 4% in the last recruitment cycle.
```

Author: Thanks! Updated.

![Screenshot showing page with the question ‘What disabilities do you have?’ and this list of options: Blind or a serious visual impairment not corrected by glasses, Dear or a serious hearing impairment, Learning difficulty for example dyslexia, Long-standing illness for example cancer, Mental health condition for example depression, Physical disability or mobility issue for example impaired use of arms or legs, Social or communication impairment for example Asperger’s and Other.](disabilities-before.png "Previous design for second question")

The advantage of asking the question ‘Are you disabled?’ on the first page is that it allowed the candidates to avoid having to read the list of checkboxes on the second page.

However this caused an issue because candidates who would have chosen some of the checkboxes may not consider themselves as disabled. This meant they would select ‘No’ on the first page which would then skip the second page. We had some evidence of this from user research sessions, where one user who was dyslexic commented that not everyone considers this a disability.

> In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority [updated their recommended list of options](https://www.hesa.ac.uk/collection/c22053/revision_history), with some wording changes and 1 new option.

Reviewer 2: It's not clear why you're telling the reader this. Presumably because it means you will need to update with the new options?

Author: Changed to:

> In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority [updated their recommended list of options](https://www.hesa.ac.uk/collection/c22053/revision_history), with some wording changes we needed to consider and 1 new option we needed to add.

> Considering the above, do you have an impairment, health condition or learning difference that has substantial and long-term impact on your ability to carry out normal day-to-day activities?

We considered that this was too much of a cognitive burden on candidates, particularly at the application stage of the process.

The [equality information pattern](https://design-system.service.gov.uk/patterns/equality-information/) in the GOV.UK Design System asks an initial question of ‘Do you have any physical or mental health conditions or illness lasting or expected to last 12 months or more?’ and then a follow-up question of ‘Do any of your conditions or illnesses reduce your ability to carry out day to day activities?’. This pattern does not include a list of disability or health conditions.

## Initial results

> Comparing the first 28 days of the 2022 to 2023 recruitment cycle to the first 28 days of the previous cycle (around 8,000 applications), 15% of candidates have selected one or more disability or health condition compared to 4% in the last recruitment cycle.

Reviewer 2: 8000 for each, or one or both?

Author: Current cycle - but last cycle was broadly similar. Have moved it to the first cycle though.
